fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -180,11 +149,11 @@
             createProperty(LDP_NAMESPACE + "constrainedBy");
     public static final Property MEMBER_SUBJECT =
             createProperty(LDP_NAMESPACE + "MemberSubject");
+    public static final Property INBOX =
```

Comment:
We should hold off on add the `INBOX` property until it is used. Currently, it is not referenced in the codebase.